Various Paper Revisions

mandli opened the pull request:
Addresses some of the comments from the editor and the first reviewer.

mandli committed:
Revise sentence suggested by editor

mandli committed:
Add text addressing citable code as requested from editor

This is a first go at this, please look at and revise.  For instance we
could add more about other ways that we do this.  There is also an
interesting discussion not yet discussed as to the likelihood that
GitHub is a long-term solution (as in decades) for the scientific
literature.

ketch commented on papers/clawpack-5x/development.tex:
```diff
@@ -193,6 +193,16 @@ \subsection{Contributing}
+interface, through the GitHub website, or a number of available applications 
+on all widely used platforms.  More importantly however is the ability to tag a
+particular version of a repository with a digital object identifier (DOI) via a
+GitHub and Zonodo\footnote{For a guide on creating a DOI to a particular
```
"via a Github and Zonodo" -> via Github and Zenodo

ketch commented on papers/clawpack-5x/development.tex:
```diff
@@ -193,6 +193,16 @@ \subsection{Contributing}
+\revised{An additional benefit of using the GitHub platform is that any version
+of the code is accessible either through the command line \texttt{git}
+interface, through the GitHub website, or a number of available applications 
+on all widely used platforms.  More importantly however is the ability to tag a
```
"importantly" -> important

ketch commented on papers/clawpack-5x/development.tex:
```diff
@@ -193,6 +193,16 @@ \subsection{Contributing}
+GitHub and Zonodo\footnote{For a guide on creating a DOI to a particular
+version of software see \url{http://guides.github.com/activities/citable-code/}}.
+The combination of these abilities address the issues of availability of
+specific versions of the software and are citable in the scientific literature.}
```
Something is grammatically off with the final sentence, but I'm not completely sure what was intended.

mandli replied:
I wanted to directly point out that these capabilities address the editor's question.  Revised to hopefully make this not awful.

mandli committed:
Minor corrections due to @ketch

ketch commented on papers/clawpack-5x/development.tex:
```diff
@@ -196,12 +196,14 @@ \subsection{Contributing}
-particular version of a repository with a digital object identifier (DOI) via a
+on all widely used platforms.  More important however is the ability to tag a
+particular version of a repository with a digital object identifier (DOI) via
 GitHub and Zonodo\footnote{For a guide on creating a DOI to a particular
```
Zonodo -> Zenodo

ketch commented:
Looks good to me after the one tiny spelling correction.

mandli committed:
Clarify parallelism in AMRClaw

mandli committed:
Add some more explanation of AMR

Includes a currently coarse image illustrating the idea

mandli committed:
Better shockbubble cell example

mandli committed:
Add better AMR cell figure

mandli committed:
Add some notes for what to mention what's changed in AMR section

mandli committed:
Add better caption and trim figure

mandli committed:
Fix minor clarification regarding operations

mandli committed:
Added PETSc reference check

mandli committed:
Address open source questions

mandli committed:
Fix discussion of parallel architectures

mandli committed:
Clarify forward references to PyClaw and AMRClaw

Added some mention that these are part of the ecosystem that comprises
Clawpack.

mandli committed:
Add some middle initials

mandli committed:
Add more rebuttal descriptions

mandli commented:
Addresses a number of issues brought up by reviewers.
